cognite/powerops/custom_modules/power_model_v1/data_models/containers/Case.container.yaml

```diff
@@ -10,19 +10,51 @@ properties:
     autoIncrement: false
     name: scenario
     description: The scenario to use when running this case
+  case:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: case
+    description: The resulting case file to send to shop after preprosessor has performed mappings on model template
+  reservoirMapping:
+    type:
+      list: false # Only one per case?
```

Comment:
I would guess it should work to have multiple (if we allow multiple cut files and multiple CUTORDER files, we should allow multiple reservoirMapping files too).